src/lib.rs

```diff
@@ -170,15 +170,23 @@ impl Tracer {
         let opq_ptr = unsafe {
             traceme_start_tracer(conf_ptr as *const TracerConf)
         };
+        if cfg!(not(travis)) {
+            // Under Travis the whole C library is stubbed and returns a NULL ptr.
```

Comment:
This comment feels like it needs to be inverted (at first I read it as "this assert is only valid on Travis")?

Reply:
It's the right way around. On travis we expect a NULL ptr. We coulsd add an else if you like.

Reply:
What I'm saying is: the comment is confusing, because the comment refers to the implict `else` block. If we say something like "We can only check for XXX when not on Travis", it might be clearer?

Reply:
I find that confusing. Shall we just kill the comment?

Reply:
As you prefer.